.github/workflows/live/afp-ids.sh

```diff
@@ -67,18 +68,73 @@ if [ $SID1CHECK = false ]; then
     echo "ERROR no alerts for sid 1"
     RES=1
 fi
+CHECK=$(jq -c 'select(.alert.signature_id == 222)' ./eve.json | wc -l)
+if [ $CHECK -ne 1 ]; then
+	echo "ERROR alerts count off for sid 222 (datasets)"
+    RES=1
+fi
+JSON=$(python3 python/bin/suricatasc -v -c "dataset-clear ipv4-list ipv4" /var/run/suricata/suricata-command.socket)
+echo $JSON
+
+sleep 5
```

Comment:
What is happening in these 5 seconds ?

Reply:
just mean to give the dataset clear to have the effect of allowing another alert for 222

Reply:
Gives food to think about the best framework for testing Suricata's features that only happen live like suricatasc...